aries-site/src/pages/index.js

```diff
@@ -32,10 +32,8 @@ const Index = () => (
             Mix, match, and stack cards to change the game
           </Heading>
           <Paragraph size="large" fill>
-            The Carte Design System was created to empower designers, developer,
-            and others to contribute in making great experiences for the
-            customer. Carte means “card game” in Italian. Pick your cards and
-            see what games you can play.
+            The Design System was created to empower designers, developer, and
+            others to contribute in making great experiences for the customer.
```

Comment:
Headline references mix and match cards, we should retain that concept (i.e. don't lose the card reference altogether).
```suggestion
            The Design System was created to empower designers, developer, and
            others to contribute in making great experiences for the customer. Pick 
            your cards, then mix, match, and stack to see what experiences you 
            can create.
```
OR
```suggestion
            The Design System was created to empower designers, developer, and
            others to contribute in making great experiences for the customer. Pick 
            your cards, then mix, match, and stack to see what games you can play.
```